EXAMPLE.md

```diff
@@ -0,0 +1,302 @@
+
+### First Boot
+
+Connect via SSH:
+```
+ssh -p 4151 root@127.0.0.1
+```
+
+Disable AppArmor right away (alternative is to configure it properly for the
+worker to give it access to things like `/usr/bin/tar`; mind that skipping
+AppArmor during installation won't help because it will be installed later as a
+dependency):
+```
+systemctl mask --now apparmor
+```
+
+### openQA Setup
+
+Install both server and worker (about 600 packages in total):
+```
+zypper install -y openQA openQA-worker
+```
+
+Setup Apache reverse proxy:
+```
+/usr/share/openqa/script/configure-web-proxy
+systemctl enable apache2
+systemctl restart apache2
+```
+
+Either disable `firewalld` through `systemctl disable firewalld` or allow access
+to TCP port 80:
+```
+firewall-cmd --zone=public --add-port=80/tcp
+firewall-cmd --zone=public --add-port=80/tcp --permanent
+```
+
+Configure authorization in `/etc/openqa/openqa.ini` (`Fake` should be fine as
+long as the server is not publicly accessible):
+```
+[auth]
+method = Fake
+```
+
+Start everything:
+```
+systemctl enable --now openqa-webui
+systemctl enable --now openqa-scheduler
+systemctl enable --now openqa-worker@1
+```
+
+At this point openQA should be accessible at <http://127.0.0.1:4150> and "Login"
+link should activate "Demo" account.
+
+Open the main menu (top-right corner) and select "Manage API keys"
+(<http://127.0.0.1:4150/api_keys>).  Remove `1234567890ABCDEF` key as it will
+expire soon and create a new one that doesn't expire.  Edit
+`/etc/openqa/client.conf` to look like this:
+```
+[localhost]
+key = INSERT_KEY_FROM_WEB_UI
+secret = INSERT_SECRET_FROM_WEB_UI
+```
+The file is read-only, so save as `:w!` in Vim.
+
+Now `openqa-cli` command should be able to send jobs, but don't do that yet
+because they will be rejected.
+
+### openQA Configuration
+
+There are still things to do in Web-UI before anything can be tested.
+If you see a specific name below, it will be referenced in some other place as
+well, but can be customized otherwise just make sure to update all occurrences.
+
+#### Manual Process
+
+Go to "Medium types" (<http://127.0.0.1:4150/admin/products>) and add a new
+installation ISO medium for QubesOS 4.2 (button is below the list):
+ - Distri: `qubesos`
+ - Version: `4.2`
+ - Flavor: `install-iso`
+ - Arch: `x86_64`
+
+Don't forget to press "Save" icon.
+
+Go to "Test suites" (<http://127.0.0.1:4150/admin/test_suites>) and add a new
+one:
+ - Name: `test-suite`
+
+Go to "Machines" (<http://127.0.0.1:4150/admin/machines>) and add a new one:
+ - Name: `qemu`
+ - Backend: `qemu`
+ - Settings:
+   ```
+   HDDSIZEGB=80
+   PART_TABLE_TYPE=mbr
+   QEMUCPU=host,+vmx
+   QEMUCPUS=2
+   QEMURAM=2048
+   QEMU_APPEND=device VGA,edid=on,xres=1024,yres=768
+   QEMU_DISABLE_SNAPSHOTS=1
+   VIRTIO_CONSOLE=1
+   ```
+
+`QEMU_APPEND` makes needles match, otherwise the screen is somewhat distorted.
+
+QubesOS/Xen needing 2 GiB of RAM is the reason for giving 6 GiB to openSUSE VM.
```

Comment:
Uhm, The [official requirements](https://www.qubes-os.org/doc/system-requirements/) says 6GB, and that's still tight for some integration tests that use several VMs. But I guess for just installation tests, it might be okay. In any case, it needs to mentioned here.